gzap.go

```diff
@@ -14,8 +16,16 @@ var Logger = getLogger()
 // logger is the package level pointer to an instantied Logger.
 var logger *zap.Logger
 
+var highPriority = zap.LevelEnablerFunc(func(lvl zapcore.Level) bool {
```

Comment:
What's the purpose of priority, here?

Reply:
The priority functions are necessary for a `zap.Core` to know when it should log a particular log. We use those functions for the zap.Cores that output to the console. We use both `stdout` and `stderr`. If a log is high priority (Error level log), we want to send it to `stderr`, if it's low priority (Info, Warn...) we send it to `stdout`.